rwengine/src/script/ScriptMachine.cpp

```diff
@@ -11,6 +11,12 @@
 #include "script/ScriptModule.hpp"
 
 void ScriptMachine::executeThread(SCMThread& t, int msPassed) {
+    // @todo check if player is respawning after death or arrest
+    if (t.isMission && t.deathOrArrestCheck && false) {
```

Comment:
This should be in conditional block.
```cpp
t.wastedOrBusted = true;
```false``` should be replaced with with (see #446)
```cpp
isWasted() || isBusted()

Reply:
Sweet! I'll include these changes next time I'm in front of a computer! :)